File: bialydunajec-angular-frontend/projects/bialydunajec-admin/src/app/camp-registrations/service/rest/response/dto/cottage-space.dto.ts
export class CottageSpaceDto {
  fullCapacity: number;
  reservations: number;
  maxFemaleTotal?: number;
  maxMaleTotal?: number;
  highSchoolRecentGraduatesCapacity?: number;
  maxFemaleHighSchoolRecentGraduates?: number;
  maxMaleHighSchoolRecentGraduates?: number;

  constructor(fullCapacity: number = null, reservations: number = null, maxFemaleTotal: number = null, maxMaleTotal: number = null, highSchoolRecentGraduatesCapacity: number = null, maxFemaleHighSchoolRecentGraduates: number = null, maxMaleHighSchoolRecentGraduates: number = null) {
    this.fullCapacity = fullCapacity;
    this.reservations = reservations;
    this.maxFemaleTotal = maxFemaleTotal;
    this.maxMaleTotal = maxMaleTotal;
    this.highSchoolRecentGraduatesCapacity = highSchoolRecentGraduatesCapacity;
    this.maxFemaleHighSchoolRecentGraduates = maxFemaleHighSchoolRecentGraduates;
    this.maxMaleHighSchoolRecentGraduates = maxMaleHighSchoolRecentGraduates;
  }
}

File: backend/bialydunajec-registrations/dto/src/main/kotlin/org/bialydunajec/registrations/dto/CottageSpaceDto.kt
package org.bialydunajec.registrations.dto

data class CottageSpaceDto(
        val fullCapacity: Int,
        val reservations: Int,
        val maxFemaleTotal: Int?,
        val maxMaleTotal: Int?,
        val highSchoolRecentGraduatesCapacity: Int?,
        val maxFemaleHighSchoolRecentGraduates: Int?,
        val maxMaleHighSchoolRecentGraduates: Int?
)
